DQMOffline/Trigger/plugins/HLTInclusiveVBFSource.cc

```diff
@@ -588,36 +588,20 @@ void HLTInclusiveVBFSource::bookHistograms(DQMStore::IBooker& iBooker, edm::Run
     std::string trigPath = "(" + v.getPath() + ")";
     iBooker.setCurrentFolder(subdirName);
 
-    MonitorElement* RECO_deltaEta_DiJet;
-    MonitorElement* RECO_deltaPhi_DiJet;
-    MonitorElement* RECO_invMass_DiJet;
-    MonitorElement* HLT_deltaEta_DiJet;
-    MonitorElement* HLT_deltaPhi_DiJet;
-    MonitorElement* HLT_invMass_DiJet;
-    MonitorElement* RECO_deltaEta_DiJet_Match;
-    MonitorElement* RECO_deltaPhi_DiJet_Match;
-    MonitorElement* RECO_invMass_DiJet_Match;
-    MonitorElement* RECOHLT_deltaEta;
-    MonitorElement* RECOHLT_deltaPhi;
-    MonitorElement* RECOHLT_invMass;
-    MonitorElement* NumberOfMatches;
-    MonitorElement* NumberOfEvents;
-
-    //dummy                     = iBooker.bookFloat("dummy");
```

Comment:
As far as I can tell, `bookFloat` doesn't have any side-effects, so the result of all these calls is unused.

Reply:
yes please remove

Reply:
@iarspider , looks like `bookFloat` calls  [bookME](https://cmssdt.cern.ch/lxr/source/DQMServices/Core/src/DQMStore.cc#0067) which has some side-effects on the store. So may be we need these `bookFloat` calls ... @cms-sw/dqm-l2 should know better  though

Reply:
> @iarspider , looks like `bookFloat` calls [bookME](https://cmssdt.cern.ch/lxr/source/DQMServices/Core/src/DQMStore.cc#0067) which has some side-effects on the store. So may be we need these `bookFloat` calls ... @cms-sw/dqm-l2 should know better though

Not in this case, I think. The `onbooking` function defaults to `NOOP` ("functor to be passed as a default argument that does not do anything."), and the `makeobject` function that is passed to bookME explicitly returns nullptr. But I agree that DQM L2s should know better.

Reply:
I think it would help if we had an example python config that uses this module, so that we can test its output and verify that those `MonitorElement`s are, indeed, booked and left empty.

Reply:
There is [one](https://github.com/cms-sw/cmssw/blob/master/DQMOffline/Trigger/test/trigger_dqmoffline_inclusiveVBFParking_cfg.py), but it's outdated